lib/xcodeproj/constants.rb

```diff
@@ -457,8 +457,8 @@ module Constants
     # @return [Hash] Possible types for a scheme's 'ExecutionAction' node
     #
     EXECUTION_ACTION_TYPE = {
-      :shell_script_action  => 'Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction',
-      :send_email_action    => 'Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.SendEmailAction',
+      :shell_script  => 'Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction',
```

Comment:
is changing these keys a breaking change?

Reply:
`EXECUTION_ACTION_TYPE ` is new and this is just me changing my mind on how to name these symbols while implementing this. So I don't think isn't a breaking change unless I'm missing something.